### PR TITLE
repo sync

### PR DESCRIPTION
**Pull request by Octomerger**
This is an automated pull request to sync changes between the public and private repos.

:robot: This pull request should be merged (not squashed) to preserve continuity across repos, so please let a bot do the merging!

### CODE REVIEW

**Comment by Bo886** [comment]
I

File: .github/workflows/update-graphql-files.yml [hunk 1]
```diff
@@ -61,9 +61,9 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           branches: graphql-schema-update
-      - if: ${{ steps.create-pull-request.outputs.pr_number }}
```
Comment (Bo886) [inline .github/workflows/update-graphql-files.yml:64]: I

File: .github/workflows/update-graphql-files.yml [hunk 2]
```diff
@@ -61,9 +61,9 @@ jobs:
...
+      - if: ${{ steps.create-pull-request.outputs.pull-request-number }}
         name: Approve
         uses: juliangruber/approve-pull-request-action@c530832d4d346c597332e20e03605aa94fa150a8
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          number: ${{ steps.create-pull-request.outputs.pr_number }}
+          number: ${{ steps.create-pull-request.outputs.pull-request-number }}
```
Comment (Bo886) [inline .github/workflows/update-graphql-files.yml:69]: P

**Review by Bo886 — COMMENTED** [review]
n

**Review by ghost — CHANGES_REQUESTED** [review]
Attach

**Pull request merged**